lib/tom_queue/deferred_work_manager_cli.rb

```diff
@@ -0,0 +1,84 @@
+$stdout.sync = true
+
+require "optparse"
+require "tom_queue"
+
+module TomQueue
+
+  class DeferredWorkManagerCLI
+    include LoggingHelper
+
+    def initialize
+      @autoload_rails = true
+      @require_paths = []
+    end
+
+    def run(argv = ARGV)
+      parse_options(argv)
+
+      load_app
+
+      deferred_manager = DeferredWorkManager.new
+      deferred_manager.start
+    end
+
+    def parse_options(args = ARGV)
+      parser = OptionParser.new do |opts|
+        opts.banner = "Usage: deferred_work_manager [options]"
+
+        opts.on("-v", "--version", "Show the version of Tom Queue") do
+          puts TomQueue::VERSION
+          exit
+        end
+
+        opts.on("--[no-]autoload_rails", "Whether to load the Rails app") do |autoload_rails|
```

Comment:
This needs more thought - right now we pull the AMQP connection parameters blindly out of the environment - so there isn't really an option _not_ to auto-load rails, so this shouldn't really be an option!